code/game/gamemodes/miniantags/morph/morph.dm

```diff
@@ -35,6 +35,7 @@
 	var/morphed = 0
 	var/atom/movable/form = null
 	var/morph_time = 0
+	var/list/blacklist_typecache
```

Comment:
var/static/list/blacklist_typecache = typecacheof(list(
the,
list,
I,
posted,
below))
```

(this is allowed in the definition as it's a ```/static```)